Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:latest
+
+RUN mkdir /chaosinventory
+RUN mkdir /data
+
+COPY . /chaosinventory/
```

Comment:
Is this tested regarding the relevant paths?
Also I'd like to keep the filestructure, in case scripts (or the sideloaded compiled frontend) will be needed.

Reply:
Whilst I haven't tested the dockerfile specifically, all of the actual application is contained in the src folder, the scripts folder only includes helper scripts (and the docker entrypoint). My main point would be the docs folder which is for sure not needed, maybe we could exclude it via a dockerignore (I do not see a value added by shipping the raw docs)?

Reply:
Ah sorry, I overlooked your new commit adressing that point :see_no_evil: